notify-send-all

```diff
@@ -120,6 +123,9 @@ main() {
 	notify-send-to)
 	    send-to "$@"
 	    ;;
+	notify-send-others)
+	    skipuser="${USER:?}"
+	    ;&
```

Comment:
Why die if if $USER is not set? There are cases where that can happen and it needn't be fatal. Maybe something like, ${USER:-`whoami`}.

Also, what's up with line 128? It's a simple fix, but now I'm wondering if you accidentally sent the untested version of your patch.

Reply:
Those cases are few, but I've added fallbacks as it's easy to do.

The `;&` is a fall-through to the next branch of the switch-statement, to avoid repeating the `send-all` command.

Reply:
Learn something new every day. I didn't know bash case statements could fall through. Thanks.